tests/cogs5e/gametrack_test.py

```diff
@@ -173,21 +174,50 @@ async def test_sb_remove(self, avrae, dhttp):
 class TestCustomCounters:
     async def test_cc_create(self, avrae, dhttp):
         avrae.message("!cc create TESTCC")
-        await dhttp.receive_message()
+        await dhttp.receive_message('Custom counter created.')
+
         avrae.message("!cc create TESTLIMITS -min 0 -max 100")
-        await dhttp.receive_message()
+        await dhttp.receive_message('Custom counter created.')
 
     async def test_cc_summary(self, avrae, dhttp):
         avrae.message("!cc")
+        char = await active_character(avrae)
+        cc_embed = EmbedWithCharacter(char)
+        # Needed to allow for embed comparison
+        char.set_setting('color', 0xffffff)
+        cc_embed.color = 0xffffff
+
+        for _ in char.consumables:
+            cc_embed.add_field(name='.+', value=r'((◉+〇*)|(\*\*Current Value\*\*:.+))(\n)*(\*\*Range\*\*: .+)*(\n)*(\*\*Resets On\*\*: .+)*')
+        await dhttp.receive_message(embed=cc_embed)
 
     async def test_cc_misc(self, avrae, dhttp):
+        char = await active_character(avrae)
+        cc_embed = EmbedWithCharacter(char)
+        # Needed to allow for embed comparison
+        char.set_setting('color', 0xffffff)
+        cc_embed.color = 0xffffff
+
+        cc_embed.add_field(name=r'.+', value=r'((\d+)|(\d+\/\d+)) (\((\+|-)\d+\))(\n)*(\(\d+ .+\))*')
         avrae.message("!cc TESTCC +5")
+        await dhttp.receive_delete()
+        await dhttp.receive_message(embed=cc_embed)
```

Comment:
it'd be cool to add an assertation on the state of the counter here and in other places!

```suggestion
        await dhttp.receive_message(embed=cc_embed)
        assert next(cc for cc in char.consumables if cc.name == "TESTCC").value == 5
```